src/components/MerchantsPage/MerchantsPage.tsx

```diff
@@ -98,30 +104,30 @@ const MerchantsPage = (props: Props) => {
             </div>
           </div>
 
-          <MerchantInfoContainer>
+          <div className={styles.merchantInfoContainer}>
             <DonationPoolBox />
 
             <div className={styles.storeInfoContainer}>
               <NavBar filterStoreType={filterStoreType} />
 
               <div className={styles.merchantsContainer}>
                 {filter.map((store: any) =>
-                  store!.seller_id !== 'send-chinatown-love' ? (
-                    <MerchantCard key={store!.seller_id} storeInfo={store} />
+                  store.seller_id !== 'send-chinatown-love' ? (
+                    <MerchantCard key={store.seller_id} storeInfo={store} />
```

Comment:
I'm pretty sure these null checks were extraneous as `.map()` will only return non-null items. Someone feel free to correct me if I'm missing context.